exopy/app/log/plugin.py

```diff
@@ -35,6 +39,16 @@ class LogPlugin(Plugin):
     #: to a handler attached to the root logger.
     gui_model = Typed(LogModel)
 
+    # Current log
+    rotating_log = Typed(DayRotatingTimeHandler)
+
+    def display_rotating(self):
+        """Display the rotating log file.
+
+        """
+        with open(self.rotating_log.path) as f:
```

Comment:
I would rather not keep the file opened longer than necessary  and would hence rather write:
```python
with open(self.rotating_log.path) as f:
    log = f.read()
LogDialog(log=log).exec_()
```